src/Tasks.UnitTests/WriteCodeFragment_Tests.cs

```diff
@@ -176,7 +221,7 @@ public void NoAttributesShouldEmitNoFile2()
         /// <summary>
         /// Bad file path
         /// </summary>
-        [Fact]
+        [WindowsOnlyFact(additionalMessage: "No invalid characters on Unix.")]
```

Comment:
Why did this test work on Unix before?

Reply:
It worked before because the path didn't exist. Now that we ensure that the path is created, the test fails because there is not an `MSB3713` error. :grinning:

Reply:
Ah, looks like 3713 is a very generic error for any failure. Makes sense, thanks!